Library/Homebrew/cmd/commands.rb

```diff
@@ -31,14 +31,18 @@ def commands
       return
     end
 
-    ohai "Built-in commands", Formatter.columns(Commands.internal_commands)
-    puts
-    ohai "Built-in developer commands", Formatter.columns(Commands.internal_developer_commands)
-
-    external_commands = Commands.external_commands
-    return if external_commands.blank?
-
-    puts
-    ohai "External commands", Formatter.columns(external_commands)
+    first = true
+
+    [["Built-in commands", Commands.internal_commands],
+     ["Built-in developer commands", Commands.internal_developer_commands],
+     ["External commands", Commands.external_commands],
+     ["Cask commands", Commands.cask_internal_commands],
+     ["External cask commands", Commands.cask_external_commands]]
```

Comment:
```suggestion
    [["Built-in commands", Commands.internal_commands],
     ["Built-in developer commands", Commands.internal_developer_commands],
     ["External commands", Commands.external_commands],
     ["Cask commands", Commands.cask_internal_commands],
     ["External cask commands", Commands.cask_external_commands]]
```

Could this be a hash instead? It looks pretty Hash-y.

Reply:
Do hashes guarantee iteration order? It works when I change it to a hash but I think an array guarantees it

Reply:
Yeh, I believe they do.